src/modules/to-do.test.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+import { localStorageSave } from './to-do.js';
+import { reassignedindex } from './functions.js';
+
+import Task from './tasks.js';
+
+jest.mock('./to-do');
+
+let todolist = [];
+let descriptioninput = '';
+function addtodolist() {
+  const index = todolist.length + 1;
+  const completed = false;
+  const newtask = new Task(completed, descriptioninput.value, index);
+  todolist.push(newtask);
+  localStorageSave();
+  descriptioninput.value = '';
+}
+
+window.removetodo = (id) => {
+  const filteredArray = todolist.filter((todo) => todo !== todolist[id - 1]);
+  reassignedindex(filteredArray);
+  return filteredArray;
+};
```

Comment:
- [ ] Your implementation of unit tests for add and remove functions is fantastic :+1:. However, it is necessary to test the functions that are inside the modules and not include them in your test file. This is because it will ensure that each function is individually tested and will identify any potential issues with the code. Therefore, please remove the functions from your test file and import them from their respective modules for accurate testing.